### DISCUSSION TITLE
how to i strip &nbsp; from "No&nbsp; "

### DISCUSSION THREAD

**Discussion by eadistar** [programble/scraper#180] [Q&A] [answered]
as title how do i strip "No&nbsp;" like to "No"

**Comment by mainrs** [comment, marked as answer]
I do not think that the crate provides the functionality as-is. You can use other crates to achieve this: https://docs.rs/html-escape/latest/html_escape/

**Comment by eadistar** [comment]
Hmm ok . Thanks
